### PR TITLE
Register CMIP5-era experiment_id entries

### LINKED ISSUE #805: Proposed CMIP5-era experiment_ids for CMIP and ScenarioMIP
```
From: Taylor, Karl E.
Date: Tuesday, October 8, 2019 at 1:29 PM
To: Brian ONeill, wgcm-wip, claudia tebaldi, detlef.vanvuuren,
Jean-Francois Lamarque, veronika.eyring, Jerry Meehl, Ron Stouffer,
Cath Senior, Greg Flato
Subject: Re: WIP telco in 18 hours

Hi All,

In additional to their ScenarioMIP runs, CCCma is carrying out some RCP scenario runs
(with CMIP5 forcing) with one of their CMIP6 CanESM models.  These simulations are
not called for in CMIP6, but can provide perspective on differences between CMIP5
and CMIP6 (forcing the same model withCMIP5 and CMIP6 future forcing).  The IPCC
will likely be interested.

I think we should enable CCCma (and anyone else) to include CMIP5 RCP future
scenario runs in the CMIP6 archive (to be run with a model that also performs the
CMIP6 DECK, historical, and at least some of the ScenarioMIP simulations).  To do
this within the CMIP6 organizational framework and software infrastructure, one
option would be to ask ScenarioMIP to "sponsor" these experiments (at tier2 or
tier3 priority), and to come up with names for these experiments (or simply adopt
the names from CMIP5:  rcp26, rcp45, rcp85, rcp60)   

What do the ScenarioMIP leaders think of this idea?

A related question concerns performing *historical* simulations with both CMIP5
forcing and CMIP6 forcing to understand how different the responses are to these
different forcing estimates.  There are a couple of options:

1) Define a new experiment name (e.g., Paul has suggested "historical-cmip5"),
and this could be considered a tier3 experiment under the "CMIP" activity, under
the responsibility of the CMIP Panel)

2) Permit publication of a CMIP5-forced historical runs under the name "historical"
and distinguish them from CMIP6-forced runs using a different "forcing_index". 
In CMIP, the forcing_index is assigned an integer value (usually, but not always,
set to "1" for "standard forcing") and is generally used to distinguish similar runs,
but with different forcing estimates (e.g., two historical runs run with different
prescribed ozone, or two historical runs with different estimates of biomass
burning aerosol precursors).  

The second option would not require any special action on our part; the first
option would require registration of 4 new experiments but would make it
absolutely clear that these runs were performed with forcing different from
the standard CMIP6 forcing (for those who can't be bothered with the definition
of "forcing_index"). 

Thoughts?

best regards,
Karl
```

### PR DESCRIPTION
**Pull request by durack1**
Fix #805

### CODE REVIEW

File: CMIP6_experiment_id.json [hunk 1]
```diff
@@ -4425,6 +4425,35 @@
...
+            "description":"CMIP5 historical experiment, using CMIP5-era forcing that concluded in 2005",
+            "end_year":"2005",
+            "experiment":"all-forcing simulation of the recent past (CMIP5-era forcing)",
+            "experiment_id":"historical-cmip5",
+            "min_number_yrs_per_sim":"156",
+            "parent_activity_id":[
+                "CMIP"
+            ],
+            "parent_experiment_id":[
+                "piControl-cmip5"
```
Comment (slavakharin) [inline CMIP6_experiment_id.json]: Should the parent be "piControl-cmip5-spinup"? In which case you will probably need an experiment id entry for  "piControl-cmip5-spinup" for which the parent would be "none".

File: CMIP6_experiment_id.json [hunk 2]
```diff
@@ -7399,6 +7428,35 @@
...
+            "description":"DECK: control (CMIP5-era forcing)",
+            "end_year":"",
+            "experiment":"pre-industrial control (CMIP5-era forcing)",
+            "experiment_id":"piControl-cmip5",
+            "min_number_yrs_per_sim":"500",
+            "parent_activity_id":[
+                "CMIP"
+            ],
+            "parent_experiment_id":[
+                "piControl-cmip5"
```
Comment (slavakharin) [inline CMIP6_experiment_id.json]: Should the parent be "piControl-cmip5-spinup"? If yes, you will probably need an experiment_id entry for "piControl-cmip5-spinup".

Comment (durack1) [reply CMIP6_experiment_id.json]: @slavakharin, good catch, updated PR should reflect this tweak

**Comment by taylor13** [comment]
Wait .... I think you forgot to add hist-nat-cmip5, hist-GHG-cmip5, and hist-aer-cmip5.

**Comment by durack1** [comment]
@taylor13 @slavakharin those `DAMIP` sims are now added

**Comment by slavakharin** [comment]
CMIP5 DAMIP experiments look ok to me.

File: CMIP6_experiment_id.json [hunk 3]
```diff
@@ -3761,6 +3790,35 @@
...
+        "hist-aer-cmip5":{
+            "activity_id":[
+                "DAMIP"
+            ],
+            "additional_allowed_model_components":[
+                "AER",
+                "CHEM",
+                "BGC"
+            ],
+            "description":"Historical anthropogenic-aerosols-only run (CMIP5-era forcing)",
```
Comment (durack1) [inline CMIP6_experiment_id.json]: description `anthropogenic-aerosols-only` vs experiment `anthropogenic aerosols-only`

Comment (slavakharin) [reply CMIP6_experiment_id.json]: For the CMIP6 counterpart it's `Historical anthropogenic-Aerosols-only run` which looks a bit funny. I don't know if we want to be consistent.

Comment (durack1) [reply CMIP6_experiment_id.json]: @slavakharin yeah that was exactly what I noticed, I am inclined to tweak the original `hist-aer` while updating this, just to correct the weird formatting

Comment (slavakharin) [reply CMIP6_experiment_id.json]: @durack1 while you at tweaking the description for these experiments, consider doing the same for `stratospheric-ozone-only` in `hist-stratO3` and `ssp245-stratO3`. Should it be `stratospheric ozone-only`? I am not sure about the correct grammar.

Comment (durack1) [reply CMIP6_experiment_id.json]: @slavakharin, for transparency I may have to open a new issue to catch these subtle tweaks, this discussion being hidden in a PR is not as clearly obvious to watchers as it should be

Comment (durack1) [reply CMIP6_experiment_id.json]: Resolved in #841 #842

File: CMIP6_experiment_id.json [hunk 4]
```diff
@@ -3732,6 +3732,35 @@
...
+            "additional_allowed_model_components":[
+                "AER",
+                "CHEM",
+                "BGC"
+            ],
+            "description":"Historical well-mixed GHG-only run. Models with interactive chemistry schemes should either turn off the chemistry or use a preindustrial climatology of stratospheric and tropospheric ozone in their radiation schemes. This will ensure that ozone is fixed in all these simulations, and simulated responses in models with and without coupled chemistry are comparable. (CMIP5-era forcing)",
+            "end_year":"2020",
+            "experiment":"historical well-mixed GHG-only run (CMIP5-era forcing)",
+            "experiment_id":"hist-GHG-cmip5",
+            "min_number_yrs_per_sim":"163",
```
Comment (slavakharin) [inline CMIP6_experiment_id.json]: Should this be 171 if the end_year=2020? In CMIP5, the end year was 2012 (and hence 163) but I prefer the current setting for end_year=2020 (171 yrs) which make them more comparable to CMIP6 runs.

Comment (taylor13) [reply CMIP6_experiment_id.json]: I don't see how the end year can be different from CMIP5.  We would need to add 8 years of CMIP5 GHG forcing to what's available.  Maybe I'm missing something.

Comment (durack1) [reply CMIP6_experiment_id.json]: @taylor13 the extension beyond the historical forcings are RCP45, and so in theory the CMIP5 simulations could have been extended through to 2100 or beyond. For this reason, there is no inconsistency to extend the `*-cmip5` experiments from 2012 to 2020

Comment (durack1) [reply CMIP6_experiment_id.json]: @slavakharin, good catch, caught that one (so it's now 171, 1850 to 2020 for all new `*-cmip5` experiments)

Comment (durack1) [reply CMIP6_experiment_id.json]: @taylor13 I think your comment is also resolved, so will resolve this thread

**Pull request merged**